st/clitests/runallsystest.sh

```diff
@@ -76,6 +76,7 @@ sh ./prechecksystest.sh
 
 # Update s3iamcli_test_config.yaml with the correct ldap info
 sed -i "s/ldappasswd:.*/ldappasswd: \'$ldap_admin_pwd\'/g" ./s3iamcli_test_config.yaml
+sed -i "s/password :.*/password : $ldap_root_pwd/g" test_data/ldap_config.yaml
```

Comment:
where are we going to access this file - ldap_config.yaml ?

Reply:
it is refered in st's it was fixed as. a part of st failures

Reply:
ok. got it.